staging/src/k8s.io/apiserver/pkg/storage/etcd3/api_object_versioner.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2020 Authors of Arktos.
```

Comment:
Is the copyright head correct?

Reply:
Got it after running make update

Reply:
I understand. You don't need to fix it right now, but those should be fixed soon.